### DISCUSSION TITLE
RSSI over -90, About GATT ERROR 133 and BleConfiguration.AndroidPauseBeforeServiceDiscovery

### DISCUSSION THREAD

**Discussion by engineearemotetechnologiessl** [shinyorg/shiny#704] [Q&A] [answered]
Hi there! This is Victor, from spain!
I'm working with shiny bluetooth ble to connect peripherals that should be over -90 rssi sometimes. As expected, connecting in that rssi range is tricky, but I try to fine tune the configuration as much as possible, play with retries and all the tools I can think of.
I was looking in the configuration and in the shiny code and I saw that BleConfiguration.AndroidPauseBeforeServiceDiscovery is not used. am I missing something? 
In case it is in the wishlist, how is it planned? 
Of course, any tricks or hints for connections at that distance would be welcome.
 
Thanks! Shiny is a great project!

**Comment by aritchie** [comment, marked as answer]
This variable is left over code that is no longer used.

**Reply by engineearemotetechnologiessl** [reply]
Thank you Allan.
May I ask why it is no longer used? Was it useless?
